discovery_engine_core/core/src/stack/ops/breaking.rs

```diff
@@ -60,16 +63,24 @@ impl Ops for BreakingNews {
     }
 
     async fn new_items(&self, _key_phrases: &[KeyPhrase]) -> Result<Vec<Article>, GenericError> {
-        if let Some(markets) = self.markets.as_ref() {
+        if let (Some(markets), Some(excluded_sources)) =
+            (self.markets.as_ref(), self.excluded_sources.as_ref())
```

Comment:
🔴  this won't fetch any new items in the case of `(Some(markets), None)`, which is a viable case we need to support. same for personalized news.